### PR TITLE
Add configuration to enable signature validation certificate alias in the console.

### PR DESCRIPTION
**Pull request by ChamathNS**
## Purpose
$subject.
fixes https://github.com/wso2/product-is/issues/11003.

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ import isEmpty from "lodash/isEmpty";
 import union from "lodash/union";
 import React, { FunctionComponent, ReactElement, useEffect, useRef, useState } from "react";
 import { useTranslation } from "react-i18next";
-import { useDispatch } from "react-redux";
+import {useDispatch, useSelector} from "react-redux";
```
Comment (DimalChandrasiri) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:29]:
```suggestion
import { useDispatch, useSelector } from "react-redux";
```

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ import {
     SAMLMetaDataInterface
 } from "../../models";
 import { CertificateFormFieldModal } from "../modals";
+import {AppState} from "../../../core";
```
Comment (DimalChandrasiri) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:40]:
```suggestion
import { AppState } from "../../../core";
```

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx [hunk 3]
```diff
@@ -637,34 +640,37 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
...
+                                                ".requestValidation.fields.signatureValidationCertAlias.label")
+                                        }
+                                        name="signatureValidationCertAlias"
+                                        type="dropdown"
+                                        required={ false }
+                                        disabled={ !isRequestSignatureValidationEnabled }
+                                        value={ initialValues?.requestValidation.signatureValidationCertAlias }
+                                        requiredErrorMessage={
+                                            t("console:develop.features.applications.forms.inboundSAML.sections" +
+                                                ".requestValidation.fields.signatureValidationCertAlias.validations.empty")
```
Comment (DimalChandrasiri) [inline apps/console/src/features/applications/components/forms/inbound-saml-form.tsx:660]: Possible character count limit exceeding. Please check and limit line character limit to 120 max.

File: apps/console/src/features/core/configs/app.ts [hunk 4]
```diff
@@ -157,6 +157,7 @@ export class Config {
             gravatarConfig: window["AppUtils"].getConfig().ui.gravatarConfig,
             isClientSecretHashEnabled: window["AppUtils"].getConfig().ui.isClientSecretHashEnabled,
             isGroupAndRoleSeparationEnabled: window["AppUtils"].getConfig().ui.isGroupAndRoleSeparationEnabled,
+            isSignatureValidationCertificateAliasEnabled: window["AppUtils"].getConfig().ui.isSignatureValidationCertificateAliasEnabled,
```
Comment (DimalChandrasiri) [inline apps/console/src/features/core/configs/app.ts:160]: Possible character count limit exceeding. Please check and limit line character limit to 120 max.

**Review by DimalChandrasiri — CHANGES_REQUESTED** [review]
Please address the comments

File: apps/console/src/features/applications/components/forms/inbound-saml-form.tsx (hunk 3 above)
Comment (ChamathNS) [reply apps/console/src/features/applications/components/forms/inbound-saml-form.tsx]: Addressed with https://github.com/wso2/identity-apps/pull/1526/commits/69aece1b3e9eb4a72d9cf5496e593dfde17523f9.

File: apps/console/src/features/core/configs/app.ts (hunk 4 above)
Comment (ChamathNS) [reply apps/console/src/features/core/configs/app.ts]: Addressed with https://github.com/wso2/identity-apps/pull/1526/commits/69aece1b3e9eb4a72d9cf5496e593dfde17523f9.

**Review by DimalChandrasiri — APPROVED** [review]

**Pull request merged**